packages/components/.storybook/buttonUtils.stories.module.css

```diff
@@ -0,0 +1,24 @@
+.recommendedVariantSet,
+.recommendedVariantSet h2,
+.recommendedVariantSet button,
+.recommendedVariantSet a {
+  @apply mb-4;
+}
+
+.arrowBackIcon {
+  margin-left: -0.4em;
+  margin-right: 0.5em;
+}
```

Comment:
you could try using [arbitrary TW values](https://tailwindcss.com/docs/just-in-time-mode#arbitrary-value-support) now 😱 
```
<ArrowBackRoundedIcon
  className="ml-[-0.4em] mr-[0.5em]"
/>
```

Reply:
Woah cool!

Reply:
I decided to leave those 2 styles in the module because it's annoying to tinker with the values when it's copied across different elements, but I updated them to use tw. Thanks for the hot tip!